Dockerfile-alpine.template

```diff
@@ -0,0 +1,44 @@
+FROM alpine:3.4
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 0.0.0
+
+RUN addgroup -S node \
+    && adduser -D -S -h /var/cache/node -s /sbin/nologin -G node node \
+    && apk add --no-cache \
+        libstdc++ \
+    && apk add --no-cache --virtual .build-deps \
+        binutils-gold \
+        curl \
+        g++ \
+        gcc \
+        gnupg \
+        libgcc \
+        linux-headers \
+        make \
+        python \
+        tar \
+    && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys \
+        9554F04D7259F04124DE6B476D5A82AC7E37093B \
+        94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+        0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
+        FD3A5288F042B6850C66B31F09FE44734EB7990E \
+        71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+        DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+        B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+        C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
```

Comment:
Can you revert this to the previous for-loop approach per my latest comment?

The for-loop allows the build to fail early because when given multiple keys, gpg will only emit a non-zreo status if all keys fail to download.